env.daint.sh

```diff
@@ -151,14 +148,13 @@ unsetCppEnvironment()
         exit 1
     esac
 
+    module unload CMake
+    module unload ncurses
```

Comment:
why do you need to unload this, it was not loaded, is there something wrong with the CMake load/unload